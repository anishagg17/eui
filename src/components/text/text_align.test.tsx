import React from 'react';
import { render } from 'enzyme';
import { requiredProps } from '../../test';

import { EuiTextAlign, TextAlignment } from './text_align';

describe('EuiTextAlign', () => {
  test('is rendered', () => {
    const component = render(<EuiTextAlign {...requiredProps} />);

    expect(component).toMatchSnapshot();
  });

  ['left', 'right', 'center'].forEach(direction => {
    test(`in ${direction} direction`, () => {
      const component = render(
        <EuiTextAlign
          {...requiredProps}
          textAlign={direction as TextAlignment}
        />
      );

      expect(component).toMatchSnapshot();
    });
  });
});
