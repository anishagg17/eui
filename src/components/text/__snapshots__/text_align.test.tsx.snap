// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EuiTextAlign in center direction 1`] = `
<div
  aria-label="aria-label"
  class="euiTextAlign euiTextAlign--center testClass1 testClass2"
  data-test-subj="test subject string"
/>
`;

exports[`EuiTextAlign in left direction 1`] = `
<div
  aria-label="aria-label"
  class="euiTextAlign euiTextAlign--left testClass1 testClass2"
  data-test-subj="test subject string"
/>
`;

exports[`EuiTextAlign in right direction 1`] = `
<div
  aria-label="aria-label"
  class="euiTextAlign euiTextAlign--right testClass1 testClass2"
  data-test-subj="test subject string"
/>
`;

exports[`EuiTextAlign is rendered 1`] = `
<div
  aria-label="aria-label"
  class="euiTextAlign euiTextAlign--left testClass1 testClass2"
  data-test-subj="test subject string"
/>
`;
